simple_live_app/lib/modules/live_room/player/player_controller.dart

```diff
@@ -48,6 +48,12 @@ mixin PlayerMixin {
   );
 }
 mixin PlayerStateMixin {
+  ///当前音量
+  final volume = 80.0.obs;
```

Comment:
media_kit音量默认值应该是100吧？
```suggestion
  final volume = 80.0.obs;
```

Reply:
不知道怎么获取音量，就自己设置了个初始值，一开始用的100，调到100后感觉又大了点就改成了80

Reply:
根据media_kit注释，默认值就是100。volume赋值100即可。

``` Dart
 /// Sets the playback volume of the [Player].
/// Defaults to `100.0`.
Future<void> setVolume(double volume) async {
  return platform?.setVolume(volume);
}
```

Reply:
已经修改了